libnopegl/src/ngpu/graphics_state.h

```diff
@@ -125,40 +125,40 @@ struct ngpu_graphics_state {
-         .read_mask  = 0xff,                                       \
-         .fail       = NGPU_STENCIL_OP_KEEP,                       \
-         .depth_fail = NGPU_STENCIL_OP_KEEP,                       \
-         .depth_pass = NGPU_STENCIL_OP_KEEP,                       \
-     },                                                            \
-    .cull_mode          = NGPU_CULL_MODE_NONE,                     \
-}                                                                  \
+    .blend              = 0,                                        \
+    .blend_src_factor   = NGPU_BLEND_FACTOR_ONE,                    \
+    .blend_dst_factor   = NGPU_BLEND_FACTOR_ZERO,                   \
+    .blend_src_factor_a = NGPU_BLEND_FACTOR_ONE,                    \
+    .blend_dst_factor_a = NGPU_BLEND_FACTOR_ZERO,                   \
+    .blend_op           = NGPU_BLEND_OP_ADD,                        \
+    .blend_op_a         = NGPU_BLEND_OP_ADD,                        \
+    .color_write_mask   = NGPU_COLOR_COMPONENT_R_BIT                \
+                        | NGPU_COLOR_COMPONENT_G_BIT                \
+                        | NGPU_COLOR_COMPONENT_B_BIT                \
+                        | NGPU_COLOR_COMPONENT_A_BIT,               \
+    .depth_test         = 0,                                        \
+    .depth_write_mask   = 1,                                        \
+    .depth_func         = NGPU_COMPARE_OP_LESS,                     \
+    .stencil_test       = 0,                                        \
+    .stencil_front      = {                                         \
+        .write_mask = 0xff,                                         \
+        .func       = NGPU_COMPARE_OP_ALWAYS,                       \
+        .ref        = 0,                                            \
+        .read_mask  = 0xff,                                         \
+        .fail       = NGPU_STENCIL_OP_KEEP,                         \
+        .depth_fail = NGPU_STENCIL_OP_KEEP,                         \
+        .depth_pass = NGPU_STENCIL_OP_KEEP,                         \
+    },                                                              \
+    .stencil_back = {                                               \
+         .write_mask = 0xff,                                        \
+         .func       = NGPU_COMPARE_OP_ALWAYS,                      \
+         .ref        = 0,                                           \
+         .read_mask  = 0xff,                                        \
+         .fail       = NGPU_STENCIL_OP_KEEP,                        \
+         .depth_fail = NGPU_STENCIL_OP_KEEP,                        \
+         .depth_pass = NGPU_STENCIL_OP_KEEP,                        \
+     },                                                             \
+    .cull_mode          = NGPU_CULL_MODE_NONE,                      \
+}                                                                   \
```

Comment:
The whitespace fix doesn't belong in this commit

Reply:
Should be fixed.